### ISSUE TITLE
Text in Mermaid graph nodes overflows boxes

### ISSUE THREAD

**Issue by jeffbarr** [dendronhq/dendron#658]
## Please select the area your Bug applys to. (Multiple selections are Possible)

- [ ] Onboard. Initial vault setup and import/export pods 
- [ ] Create. Note creation, lookup, snippets and templates
- [ ] Retrieve. Backlinks, references, graph view
- [ ] Structure. Refactoring, multi-vault and schemas
- [ ] Publish. Sharing your repo with the world
- [X] Misc (Choose this if your not sure)

## Describe the bug

Text in a Mermaid graph overflows the box.

## To Reproduce

1. Go to a Markdown page
2. Enter the graph below
3. Open the Dendron preview
4. See that the text overflows the box

## Expected behavior

Text fits inside the box

## Sample graph

```mermaid
graph TD;
    A[Daily]-->B[Personal TODO];
    A-->C[Work TODO];
    A-->D[Calendar];
    A-->E[LinkedIn];
```
## Screenshots

![dd_mermaid_overflow](https://user-images.githubusercontent.com/78832/115641106-45ebaa80-a2cd-11eb-84ff-e92e0722658f.png)

## Desktop (please complete the following information)

- OS: [Windows 10]

## Dendron Log file

Please attach the output of `>Dev:Dendron: Open Logs` here

**Comment by kpathakota** [comment]
This will likely get improved with the changes required for #643 but for now, it's in a component that we only tangentially own (markdown previewer). We should do a check once 643 is closed though.

**Comment by micharris42** [comment]
The text overflow issues appears to have been fixed since Dendron switched to its own preview. 

Should this issue be marked as Closed?

**Comment by kevinslin** [comment]
Yep. The only issue is that preview v2 doesn't currently support export to pdf which I think @jeffbarr might need. Will keep this open until we have https://github.com/dendronhq/dendron/issues/1204 implemented
